core/reporter-impl/src/test/java/org/arquillian/reporter/impl/asserts/ReportAssert.java

```diff
@@ -151,6 +149,35 @@ public REPORTASSERTTYPE hasEntriesContaining(List<Entry> expectedEntries) {
         return (REPORTASSERTTYPE) this;
     }
 
+    public REPORTASSERTTYPE hasEntryContainingKeys(StringKey... expectedStringKey) {
+        return hasEntryContainingKeys(Arrays.asList(expectedStringKey));
+    }
+
+    public REPORTASSERTTYPE hasEntryContainingKeys(List<StringKey> expectedStringKeys) {
+        isNotNull();
+
+        expectedStringKeys.parallelStream().forEach(expectedStringKey -> {
+            assertThat(getReportKeys(actual))
```

Comment:
This is inside of the loop - which means that the whole method is performed several times, which is not needed - assign it to a local variable before the loop is executed and then just reference the local variable.